[13.0][MIG] sale_order_price_recalculation (#1155)
OCA/sale-workflow: sale_order_price_recalculation/readme/DESCRIPTION.rst

> This module add 2 buttons on sale orders (below sale order lines) that:

Reviewer: ```suggestion
This module adds 2 buttons on sale orders (below sale order lines) that:
```

> * Recalculates the prices of the order lines that contain a product in them.

Reviewer: ```suggestion
* Recalculate the prices of the order lines that contain a product
```

> * Reset product descriptions from current product information.

Reviewer: ```suggestion
* Reset product descriptions using current product information
```
